python/golem_task_api/structs.py

```diff
@@ -1,6 +1,15 @@
-from typing import Any, Dict, List
+from typing import Any, Dict, List, Optional
 
-from dataclasses import dataclass
+from dataclasses import dataclass, field
+
+
+@dataclass
+class Infrastructure:
+    @dataclass
+    class Memory:
```

Comment:
This seems a little bit of an overkill. I'd just name the field `min_mem_gib`. But I don't insist if you believe this is the proper way to handle this.

Reply:
You're right, it makes no sense for a single property